pkg/domain/entities/images.go

```diff
@@ -377,3 +377,7 @@ type ImageUnmountReport struct {
 	Err error
 	Id  string // nolint
 }
+
+type ImageScanOptions struct {
```

Comment:
Please add comments to the struct and its fields.

Not all types in the `entities` package are commented yet but they're super helpful in IDEs and docs.